### PR TITLE
SO-5544: Log errors encountered during remote job execution (8.x) 

### PR DESCRIPTION
**Pull request by nagyo**

### CODE REVIEW

**Review by cmark — APPROVED** [review]
🥓

File: core/com.b2international.snowowl.core/src/com/b2international/snowowl/core/jobs/RemoteJob.java [hunk 1]
```diff
@@ -119,6 +123,7 @@ protected final IStatus run(IProgressMonitor monitor) {
 					.build();
 			}
 			
+			LOG.error("Error encountered while executing remote job", e);
```
Comment (apeteri) [inline core/com.b2international.snowowl.core/src/com/b2international/snowowl/core/jobs/RemoteJob.java]: Some identifying information about the job should be added to the message. `id`, `description`, class name of `request` are all good candidates.

**Review by apeteri — CHANGES_REQUESTED** [review]

File: core/com.b2international.snowowl.core/src/com/b2international/snowowl/core/jobs/RemoteJob.java (hunk 1 above)
Comment (nagyo) [reply core/com.b2international.snowowl.core/src/com/b2international/snowowl/core/jobs/RemoteJob.java]: +1 for that

**Review by apeteri — APPROVED** [review]
👍

**Pull request merged**